src/game/input.c

```diff
@@ -2,16 +2,37 @@
 
 #include "specific/s_input.h"
 
+#define DELAY_FRAMES 12
+#define HOLD_FRAMES 3
```

Comment:
OK now it's readable, we wait 12 frames till letting the user cycle fast, and then we only do it once every 3 frames.

Reply:
Ya. Then we can just scale this for 60 FPS in the future.